### PR TITLE
[No JIRA] Remove redundant dependency

### PR DESCRIPTION
**Pull request by k0nserv**
Remember to include the following changes:
+ [x] `UNRELEASED.yaml`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -33092,7 +33023,7 @@
       "dependencies": {
         "got": {
           "version": "6.7.1",
-          "resolved": "https://registry.npmjs.org/got/-/got-6.7.1.tgz",
+          "resolved": "http://registry.npmjs.org/got/-/got-6.7.1.tgz",
```
Comment (georgegillams) [inline package-lock.json]: Could you revert these changes?

Comment (k0nserv) [reply package-lock.json]: They happen when I run `npm install`, which I need to do to remove the dependency

Comment (k0nserv) [reply package-lock.json]: Let me try an alterantive approach

Comment (k0nserv) [reply package-lock.json]: This is just `npm` being `npm` I think.

Here's my env

```
$ npm --version
6.13.4

$ node --version
v12.14.1
```

I ran `npm  uninstall -D static-site-generator-webpack-plugin`

Comment (georgegillams) [reply package-lock.json]: Yeah there's no way to stop NPM being dumb, but we can still be selective about what we check in.
It's not a massive issue, but it makes our git history messier if we always let NPM change stuff back and forth

Comment (k0nserv) [reply package-lock.json]: I'm weary of manually editing `package-lock.json` though
